crates/sui-node/src/main.rs

```diff
@@ -73,6 +71,8 @@ fn main() {
         .with_prom_registry(&prometheus_registry)
         .init();
 
+    drop(metrics_rt);
+
```

Comment:
Are these changes necessary?

Reply:
Yes - otherwise when we initialize TelemetryConfig, there is no active tokio runtime, so it can't install the batch exporter.